openfisca-web-api/endpoints.md

```diff
@@ -4,5 +4,6 @@ Each OpenFisca Country Package web API comes with a set of endpoints including a
 You can check out the `OpenFisca-France` [swagger documentation](https://fr.openfisca.org/legislation/swagger) to see how their endpoints work.
 
 The Openfisca Web API can be used to:
- - access information about the parameters (e.g. `/parameters`) and variables (e.g. `/variables`) of the Country Package.
+ - access information about the parameters (e.g. `/parameters`) and variables (e.g. `/variables`) of the Country Package,
+ - access information about how persons are defined in the Country Package (e.g. `/entities`),
```

Comment:
Why not include this in the previous line? Having a single line for `/entities` make me feel like this route is particularly important, while it's probably at the same level that `/variables` and `parameters`